grr/server/grr_response_server/flows/general/osquery_test.py

```diff
@@ -169,7 +164,7 @@ def setUp(self):
     super(FakeOsqueryFlowTest, self).setUp()
     self.client_id = self.SetupClient(0)
 
-  def _NewTestFlowStartAndRun(self, query):
+  def _InitializeFlow(self, query):
```

Comment:
`_InitializeFlow` and `_RunQuery` are very much alike (except that the other one also collects results). How about renaming them to `_InitializeOsqueryFlow` and `_RunOsqueryFlow`? Then both have consistent naming hinting similar responsibilities (with the second one doing a bit more).

We might want to drop the `Osquery` part (since it is kind of obvious in tests for osquery), then `_RunQuery` would just become `_RunFlow`.

What do you think?